feat(kiosk): add kiosk-mode

Yasamato opened the pull request:
To enable kiosk-mode run with `--kiosk` argument

Yasamato committed:
feat(kiosk): add kiosk-mode

To enable kiosk-mode run with `--kiosk` argument

Agent77326 commented:
TODO:
- add kiosk-mode for computer-selection in the game-creation screen.
- get a computer-jar to be run as computer

Yasamato committed:
feat(kiosk): add default com-player

Yasamato committed:
feat(kiosk): add stop-game button

Agent77326 commented:
Kiosk-mode does not work in compiled-version.

Yasamato committed:
chore(dist): add option to enable kiosk-dist

Added an option in main.js to enable electron-builder to buil an dist where kiosk-mode is permanently enabled.

Yasamato committed:
chore(dependencies): bump version

The updated packages do not break functionality of our code (at least on win).

For more information about the issue with electron update see: https://github.com/CAU-Kiel-Tech-Inf/socha-gui/issues/39

Agent77326 commented:
The issue with not being able to pass arguments on compiled version will be eventually addressed in a different issue. See #108
